src/util-napatech.c

```diff
@@ -1454,12 +1455,17 @@ uint32_t NapatechSetupTraffic(uint32_t first_stream, uint32_t last_stream)
             if (strchr(port->val, '-')) {
                 stream_spec = CONFIG_SPECIFIER_RANGE;
 
-                ByteExtractStringUint8(&ports_spec.first[iteration], 10, 0, port->val);
-                ByteExtractStringUint8(&ports_spec.second[iteration], 10, 0, strchr(port->val, '-')+1);
+                if (ByteExtractStringUint8(&ports_spec.first[iteration], 10, 0, port->val) == -1) {
+                    SCLogError("Invalid value in napatech.ports specification in conf file.");
```

Comment:
error w/o consequence should be a warning, alternatively truly error out here

Reply:
pattern repeats multiple times

Reply:
SCLogError() are replaced with SCLogWarning() in the latest commit.